### DISCUSSION TITLE
Composer error for spatie/laravel-ray

### DISCUSSION THREAD

**Discussion by OgaBoss** [spatie/ray#140] [Q&A]
When installing the spatie/laravel-ray via composer i get this error

Problem 1
> 1 Root composer.json requires spatie/laravel-ray 1.4.0 -> satisfiable by spatie/laravel-ray[1.4.0].
> 2 `spatie/laravel-ray` 1.4.0 requires illuminate/database ^7.0|^8.13 -> found illuminate/database[v7.0.0, ..., 7.x-dev, v8.13.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

My project is on laravel 8.12.

Thanks.

**Comment by sinnbeck** [comment]
Try posting your composer.json file
